metadata.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "puppet-pulpcore",
+  "version": "0.1.0",
+  "author": "William Clark",
```

This should match the user we're using to publish:
```suggestion
  "author": "theforeman",
```